pepper-apis/src/main/java/org/broadinstitute/ddp/service/UserService.java

```diff
@@ -73,9 +73,9 @@ public void deleteUser(Handle handle, User user) throws IOException {
         mailAddressDao.deleteAddressByParticipantId(userId);
         // Removing all the answers and activity instances
```

Comment:
This is used by one of our APIs for deleting a user that meets certain criteria. I went ahead and updated this to handle nested instances as well.